### ISSUE TITLE
Attempt to publish all the dataset in one go resulted in 504 error

### ISSUE THREAD

**Issue by ManonGros** [gbif/ipt#1526]
We got this message from an IPT administrator.
He said that he got the error after 45min or so. He is asking if this could be a memory issue and if this is safe to restart TomCat.
He mentions that the IPT logs seem to suggest the system has hung, as no new progress is added.

**Comment by tvandenberghe** [comment]
We have restarted tomcat, and after that republished al datasets separately. The reason is due to one single large dataset (600000 occurrences and a few million extendedmeasurementorfacts), that can't be published in isolation as well (server hangs and no indication of progress). If i extrapolate the IPT building datasets of 50000 records, it shouldn't take more than 10 minutes, so I think it's a server memory issue. Around a year ago, we published this dataset successfully, it had around 500000 records back then. How much memory is needed for such large datasets?
